src/Toolkit/Date.php

```diff
@@ -264,9 +248,8 @@ public function month(int|null $month = null): int
 	 * Returns the datetime which is nearest to the object's datetime
 	 *
 	 * @param string|int|\DateTimeInterface ...$datetime Datetime strings, UNIX timestamps or objects
-	 * @return string|int|\DateTimeInterface
 	 */
-	public function nearest(...$datetime)
+	public function nearest(...$datetime): string|int|DateTimeInterface
```

Comment:
Is there any harm in using it like this?

```suggestion
	public function nearest(string|int|DateTimeInterface ...$datetime): string|int|DateTimeInterface
```